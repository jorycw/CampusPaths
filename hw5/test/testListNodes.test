# Testing file.
# Tests listing back the nodes in graph, both alphabetically and 
# with an empty graph

# Create graph
CreateGraph graph1

# Lists node of an empty graph
ListNodes graph1

# Adds 3 different nodes to graph1
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Lists nodes that are easily alphabetized of graph1
ListNodes graph1

# Add 5 more nodes to graph1 that are close in string comparison
AddNode graph1 n
AddNode graph1 nn
AddNode graph1 4n
AddNode graph1 m
AddNode graph1 o

# Lists nodes checking for order of nodes alphabetically of graph1
ListNodes graph1