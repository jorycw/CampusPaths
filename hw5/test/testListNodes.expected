# Testing file.
# Tests listing back the nodes in graph, both alphabetically and 
# with an empty graph

# Create graph
created graph graph1

# Lists node of an empty graph
graph1 contains:

# Adds 3 different nodes to graph1
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Lists nodes that are easily alphabetized of graph1
graph1 contains: n1 n2 n3

# Add 5 more nodes to graph1 that are close in string comparison
added node n to graph1
added node nn to graph1
added node 4n to graph1
added node m to graph1
added node o to graph1

# Lists nodes checking for order of nodes alphabetically of graph1
graph1 contains: 4n m n n1 n2 n3 nn o